tests/test_postgresql.py

```diff
@@ -1156,6 +1157,61 @@ def test_inline_exclude_constraint_case_sensitive(self):
             "name='TExclX'))",
         )
 
+    def test_inline_exclude_constraint_literal_column(self):
+        # Issue: https://github.com/sqlalchemy/alembic/issues/1184
```

Comment:
**mike bayer** (zzzeek) wrote:

it would be preferred to label this like:

   
    """fix for #1184"""

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/4493